stdlib/public/core/LazySequence.swift

```diff
@@ -68,25 +68,25 @@
 ///           return result
 ///         }
 ///       }
-///       private var nextElement: ResultElement? // The next result of next().
-///       private var base: Base                  // The underlying iterator.
-///       private let nextPartialResult: (ResultElement, Base.Element) -> ResultElement
+///       var nextElement: ResultElement? // The next result of next().
+///       var base: Base                  // The underlying iterator.
+///       let nextPartialResult: (ResultElement, Base.Element) -> ResultElement
 ///     }
 ///     
 ///     struct LazyScanSequence<Base: Sequence, ResultElement>
 ///       : LazySequenceProtocol // Chained operations on self are lazy, too
 ///     {
 ///       func makeIterator() -> LazyScanIterator<Base.Iterator, ResultElement> {
 ///         return LazyScanIterator(
-///           nextElement: initial, base: base.makeIterator(), nextPartialResult)
+///           nextElement: initial, base: base.makeIterator(), nextPartialResult: nextPartialResult)
 ///       }
-///       private let initial: ResultElement
-///       private let base: Base
-///       private let nextPartialResult:
+///       let initial: ResultElement
+///       let base: Base
+///       let nextPartialResult:
 ///         (ResultElement, Base.Element) -> ResultElement
 ///     }
 ///
-/// and finally, we can give all lazy sequences a lazy `scan` method:
+/// // and finally, we can give all lazy sequences a lazy `scan` method:
```

Comment:
```suggestion
/// and finally, we can give all lazy sequences a lazy `scan` method:
```

This is correct as it was. (Code voice is automatically applied based on indenting.)

Reply:
Sorry, was trying to make it easier to copy and paste the code into a playground and have it run with fewer steps.  Thanks for catching this (!).